eZ/Publish/Core/Persistence/Legacy/Content/Type/Handler.php

```diff
@@ -619,4 +619,23 @@ public function publish( $contentTypeId )
 
         $this->updateHandler->publishNewType( $toType, Type::STATUS_DEFINED );
     }
+
+    /**
+     * @see \eZ\Publish\SPI\Persistence\Content\Type\Handler::getFieldMap
+     */
+    public function getFieldMap()
+    {
+        $fieldMap = [];
```

Comment:
It does run them. But after we have merged to the stable branch. I'd rather
keep them stable, and be warned earlier :-)

One ways is to also send pull requests for backports, but I don't think we
"always" do it, do we ? P.S. We could take that to another channel :)